src/faldbt/project.py

```diff
@@ -190,6 +192,8 @@ class FalDbt:
 
     _firestore_client: Union[FirestoreClient, None]
 
+    _el_configs: Dict[str, Any]
```

Comment:
Proposal: create a class AirbyteProfileConfig and FivetranProfileConfig and write the behavior of `_run_el_sync` in them. That way calling `fivetran_sync(name, ...)` is like

```py
def fivetran_sync(name, ...)
  profile_config = self._el_configs.get(name)
  if profile_config is None:
    raise
  profile_config.sync(...)
```